src/test/swift/com/sleekbyte/tailor/functional/CommaWhitespaceTest.swift

```diff
@@ -207,3 +207,49 @@ enum ASCIIControlCharacter: Character {
     case Tab = "\t",LineFeed = "\n"
     case CarriageReturn = "\r"
 }
+
+var (x, y): (Int, Int)
+
+var (x,y): (Int , Int)
+
+var (x, y ,z): (Int,Int, Int)
+
+var (x): Int = 2
+
```

Comment:
For the sake of more test cases, can you add a case where a tuple is returned by a function?

```swift

1
2
3
4
5
6
7
func getFullName() -> (first: String, last: String) {
    let firstName = "Aditya"
    let lastName = "Trivedi"
    
    return (firstName, lastName)
}
```